server/plugin.ts

```diff
@@ -12,38 +12,59 @@ import {
   Logger,
 } from '../../../src/core/server';
 
-import { CustomImportMapPluginSetup, CustomImportMapPluginStart } from './types';
+import { HomeServerPluginSetup } from '../../../src/plugins/home/server';
+import { getFlightsSavedObjects } from './services/sample_data/flights_saved_objects';
+
+import {
+  AppPluginSetupDependencies,
+  CustomImportMapPluginSetup,
+  CustomImportMapPluginStart,
+} from './types';
 import { createGeospatialCluster } from './clusters';
 import { GeospatialService, OpensearchService } from './services';
 import { geospatial, opensearch } from '../server/routes';
 import { mapSavedObjectsType } from './saved_objects';
 import { capabilitiesProvider } from './saved_objects/capabilities_provider';
+import { ConfigSchema } from '../common/config';
 
 export class CustomImportMapPlugin
   implements Plugin<CustomImportMapPluginSetup, CustomImportMapPluginStart> {
   private readonly logger: Logger;
   private readonly globalConfig$;
+  private readonly config$;
 
   constructor(initializerContext: PluginInitializerContext) {
     this.logger = initializerContext.logger.get();
     this.globalConfig$ = initializerContext.config.legacy.globalConfig$;
+    this.config$ = initializerContext.config.create<ConfigSchema>();
```

Comment:
what does adding $ at end means?

Reply:
I think it is the common naming convention for observable variables
https://stevenschwenke.de/DollarSignsInVariableNamesForObservablesInTypeScript